### PR TITLE
added a Tennis game

### PR DESCRIPTION
**Pull request by thanmaisai**
# Zero To Mastery - Webblocks 2022 📦📦

## PR contains:
- Added a Tennis game which is created using html css and javascript.

## Breaking changes
- None

## Screenshots
Screenshot 2022-10-12 at 5 50 25 PM
Screenshot 2022-10-12 at 5 50 18 PM

### CODE REVIEW

**Review by MattCSmith — CHANGES_REQUESTED** [review]
Hey, 
Looks good so far but there are a couple of things required here:
- Your work should be in the block directory
- You need a meta.json file
- You need an image file

Check the README.md file fore more information.

File: Tennis/index.html [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+            if(bally < 0){
+                ballspeedy = -ballspeedy ;
+            }
+
+            // bounce the ball from the right wall
+            if(bally > canvas.height){
+                ballspeedy = -ballspeedy ;
+            }
+        }
+        // function coloRect(leftX,topY,width,height,drawColor){
```
Comment (sophiabrandt) [inline Tennis/index.html:181]: Please remove outdated code.

File: Tennis/index.html [hunk 2]
```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
```
Comment (sophiabrandt) [inline Tennis/index.html:1]: This file is in the wrong location. Please make a folder with your username in `src/blocks` and move it there.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
Screenshot and `meta.json` missing. Please go to the guide in the `README` and follow all steps to create a valid contribution.

**Review by MattCSmith — CHANGES_REQUESTED** [review]
This file and directory still need removing/moving:



image

Otherwise, it looks good to me!

**Comment by MattCSmith** [comment]
Closing this PR due to unresolved change request. Feel free to resolve and reopen

**Pull request closed without merging**